CleanCode.cpp

```diff
@@ -0,0 +1,18 @@
+#include <iostream>
+#include "ColorPair.h"
+#include "ColorPairEnum.h"
+#include "testColorPair.h"
+
+
+
+int main()
```

Comment:
Hard to guess that the entry point of the functionality is in a file called CleanCode.cpp

Reply:
Hi , Thank you for the feedback.
I have made all the required.